app/src/main/java/com/ucsd/cse110t15m260/roommateinventory/ApartmentFragment.java

```diff
@@ -122,46 +227,92 @@ public void done(ParseException e) {
     }
 
     @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+
+        if (REQUEST_ITEM_CODE == requestCode) {
+            if(resultCode == AddItemActivity.RESULT_OK); {
+                //get the item
+                String itemId = (String) data.getCharSequenceExtra("item");
+                InventoryItem item = InventoryItem.getInventoryItemById(itemId);
+
+                if (item == null) {
+                    Log.d("Failed to send notif", "Item creation failed");
+                    return;
+                }
+
+                Person person = ApartmentManager.apartmentManager.getCurrentApartment().getMembers().get(personPosition);
+                PushNotifsManager.getInstance().sendToUser(person, item);
+                showCompletedNotif(person);
+            }
+        }
+    }
+
+    public void showCompletedNotif(Person person) {
+        Toast.makeText(
+                this.getContext(),
+                "Notification sent to " + person.getName(),
+                Toast.LENGTH_SHORT
+        ).show();
+    }
+
+    @Override
     public void onResume() {
         super.onResume();
 
-        Person person = AccountManager.accountManager.getCurrentUser();
-        Apartment currApt = ApartmentManager.apartmentManager.getCurrentApartment();
-
+        Person person = Person.getCurrentPerson();
+        Apartment currApt = person.getApartment();
+        mPeople.clear();
         if (person != null && person.hasApartment()) {
             for(Person mate: currApt.getMembers()) {
                 mPeople.add(mate.getName());
             }
+
+            updateUI(person);
         }
-        updateUI(person);
     }
 
-    void updateUI(Person person)
-    {
+    void updateUI(Person person) {
```

Comment:
@Satre95 What's with this weird diff? Should this change have introduced all this code?